airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorRegistrySourceDefinition.py

```diff
@@ -87,6 +117,11 @@ class BreakingChangeScope(BaseModel):
     __root__: StreamBreakingChangeScope = Field(..., description="A scope that can be used to limit the impact of a breaking change.")
 
 
+class GeneratedFields(BaseModel):
```

Comment:
As I said in https://github.com/airbytehq/airbyte/pull/37802 I think all generated fields should stay in the registry, not be metadata fields (which is what you're doing here 👍 )